content/home-2020-oct/card_12/en-US.md

```diff
@@ -0,0 +1,13 @@
+---
+# card 18
```

Comment:
card 12

(I don't know why we put the card numbers in the YAML, this convention started before me and causes me exactly the same problem(

Reply:
Heh. Thanks for catching these. 🙈

Reply:
> (I don't know why we put the card numbers in the YAML, this convention started before me and causes me exactly the same problem(

I'm pretty sure it's my fault. My script that I wrote to extract the data out of the existing page and into the first set of these files added it so I could see it easily in my editor since all the file names were just `en-US.md`. It's definitely not a requirement for them to stay.

Reply:
Thanks for the history @pmac. I might drop them in my next update.